nuxeo-core/nuxeo-core-schema/src/main/java/org/nuxeo/ecm/core/schema/SchemaManagerImpl.java

```diff
@@ -924,4 +931,39 @@ public boolean getAllowVersionWriteForDublinCore() {
         return allowVersionWriteForDublinCore;
     }
 
+    /*
+     * ===== Property API =====
+     */
+
+    /**
+     * @since 11.1
+     */
+    protected synchronized void registerProperty(List<PropertyDescriptor> descriptors) {
+        securedProperties = descriptors.stream()
+                                       .filter(PropertyDescriptor::isSecured)
+                                       .map(descriptor -> computePropertyKey(descriptor.schema, descriptor.name))
+                                       .collect(Collectors.toSet());
+    }
+
+    /**
+     * @since 11.1
+     */
+    protected synchronized void clearProperty() {
+        securedProperties.clear();
+    }
+
+    /**
+     * @since 11.1
+     */
+    @Override
+    public boolean isSecured(String schema, String path) {
```

Comment:
This is actually unused right? But I understand it can be useful.

Reply:
OK, it's because `SchemaManager` extends `PropertyCharacteristicHandler`. But why does it in fact?

Reply:
It is used in property classes, like `AbstractProperty`.